python/TestHarness/JobDAG.py

```diff
@@ -166,8 +166,8 @@ def _doPreviouslyFailed(self, job):
             tester.addCaveats('re-running')
             return
 
-        # This job was skipped, passed or silent
-        elif status in job.job_status.getSuccessStatuses():
+        # This job was skipped, passed, pending or silent
+        elif status in job.job_status.getSuccessStatuses()+job.job_status.getPendingStatuses():
```

Comment:
```suggestion
        elif status in job.job_status.getSuccessStatuses() or status in job.job_status.getPendingStatuses():
```

Reply:
Is there a meaningful difference to this? They both work and list concat is super quick in python, so I'm not even sure the suggestion is faster.

Reply:
Also important to note that these are just getting the enums for the error codes, and the total list is like size 8. Pending statuses is a 3 size list and success statuses is like 5 or 6. Obviously the list concat is bad practice on a larger/dynamic list but here I think its fine and appropriate

Reply:
Important to note that I also originally wrote it the way the commit suggests, then realized there was an equally pythonic way to do it that would also look cleaner and noticed no performance loss from the change.

Reply:
It's just a preference; avoids construction of temporaries and doesn't read any better/worse